chainer/functions/connection/n_step_lstm.py

```diff
@@ -373,8 +374,10 @@ def _stack_weight(ws):
 
 
 def n_step_lstm(
-        n_layers, dropout_ratio, hx, cx, ws, bs, xs):
-    """Stacked Long Short-Term Memory function for sequence inputs.
+        n_layers, dropout_ratio, hx, cx, ws, bs, xs, **kwargs):
+    """n_step_lstm(n_layers, dropout_ratio, hx, cx, ws, bs, xs)
```

Comment:
Sorry, I just did not know we can overwrite the API in the document when we write the API in the first line. I agree with we do not show `kwargs` in the document.